app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/LectureViewColumnAdapter.kt

```diff
@@ -0,0 +1,39 @@
+package nerd.tuxmobil.fahrplan.congress.schedule
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import nerd.tuxmobil.fahrplan.congress.R
+import nerd.tuxmobil.fahrplan.congress.models.Lecture
+
+internal interface LectureViewEventsHandler : View.OnCreateContextMenuListener, View.OnClickListener
+
+internal class LectureViewColumnAdapter(
+        private val lectures: List<Lecture>,
+        private val lectureLayoutParams: Map<Lecture, LinearLayout.LayoutParams>,
```

Comment:
:abc: Consistency: I would like to come up with a consistent naming schema for map variables like this. In `FahrplanFragment#addRoomColumns` you are using `per` as the binding word in `lecturesPerRoom`. I have used `by` or `to` myself to verbalize the key value association. Kotlin uses the `by` suffix in the [`associateBy`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/associate-by.html) function. What do you think is a good word to consistently use in such cases as `lectureLayoutParams`?

Reply:
Im going with by :)